server/graphql/v2/query/collection/AccountsCollectionQuery.ts

```diff
@@ -21,6 +22,11 @@ const AccountsCollectionQuery = {
       type: new GraphQLList(GraphQLString),
       description: 'Only accounts that match these tags',
     },
+    tagSearchOperator: {
+      type: TagSearchOperator,
```

Comment:
`null` wouldn't be a valid value, and we have a default. We can safely say:

```suggestion
      type: new GraphQLNonNull(TagSearchOperator),
```